yang/vyatta-protocols-frr-link-params-v1.yang

```diff
@@ -0,0 +1,229 @@
+				type uint32 {
+					range "1..max";
+				}
+				configd:help "Link metric for MPLS-TE purpose";
+				description "Link metric for MPLS-TE purpose";
+			}
+			leaf neighbor {
+				type types:ipv4-address;
+				configd:help "Configure remote ASBR information (Neighbor IP address and AS number)";
+				description "Configure remote ASBR information (Neighbor IP address and AS number)";
+			}
+			leaf packet-loss {
+				type uint32 {
+					range 0..100;
+				}
+				units "Percentage";
+				configd:help "Unidirectional Link Packet Loss (Percentage)";
+				description "Unidirectional Link Packet Loss (Percentage)";
+			}
+			leaf residual-bandwidth {
+				type uint64;
+				units "Bytes/second";
+				configd:help "Unidirectional Residual Bandwidth (Bytes/second)";
+				description "Unidirectional Residual Bandwidth. For a link or forwarding adjacency, residual
+					bandwidth is defined to be Maximum Bandwidth [RFC3630] minus the bandwidth currently allocated
+					to RSVP-TE LSPs. For a bundled link, residual bandwidth is defined to be the sum of the
+					component link residual bandwidths [draft-ietf-isis-te-metrics-extension-03.txt].";
+				must "../max-bandwidth" {
+					error-message "Maximum Bandwidth must be set";
+				}
+				must "../residual-bandwidth <= ../max-bandwidth" {
+					error-message "Residual Bandwidth cannot be greater than Maximum Bandwidth";
+				}
+			}
+			container unreserved-bandwidth {
+                                configd:help "Unreserved bandwidth at each priority level";
+                                description "Unreserved bandwidth at each priority level. Unreserved Bandwidth [RFC3630], is
+					subtracted from the Maximum Reservable Bandwidth (the bandwidth that can theoretically
+					be reserved) [RFC3630] and provides per-QoS-class [draft-ietf-isis-te-metrics-extension-03]";
+                                list priority {
+                                        configd:help "Priority";
+                                        description "8 sets of priority levels";
+                                        key "tagnode";
+                                        leaf tagnode {
+                                                type uint32 {
+                                                        range "0..7";
+                                                }
+                                                configd:help "Priority";
+                                                description "8 sets of priority levels";
+                                        }
+                                        leaf bandwidth {
+                                                type uint64;
+                                                units "Bytes/second";
+                                                configd:help "Bandwidth (Bytes/second)";
+                                                description "Bandwidth (Bytes/second) for each priority levels";
+						must "../../../max-bandwidth" {
```

Comment:
bad alignment

Reply:
It has been aligned

Reply:
The changes have been reflected in the last commit